### ISSUE TITLE
Address angular warnings regarding missing aria-label

### ISSUE THREAD

**Issue by milesap** [fgpv-vpgf/fgpv-vpgf#1700]
These warnings are currently suppressed in `app.module.js` as `$mdAriaProvider.disableWarnings();`. We should address these and remove this log suppression. 

Angular seems to be logging warnings even when aria labels are present as seen in this screenshot:

![waytogoangular](https://cloud.githubusercontent.com/assets/10187181/23030596/376ab5da-f43c-11e6-818d-7c28137b2407.png)
